package.json

```diff
@@ -8,35 +8,32 @@
   "license": "LGPL-3.0-or-later",
   "scripts": {
     "build": "tsc",
-    "test": "mocha build/test/**/*.test.js",
+    "test": "yarn jest",
```

Comment:
Just `jest` is enough. The scripts could be run from eg. `npm` without yarn installed